### DISCUSSION TITLE
Exercise to create website on wordpress

### DISCUSSION THREAD

**Discussion by Cyril-Mathew** [github/india#399] [Students]
An error occurs when I am trying to review and create, I am pasting a screenshot for the same. Please help me with this. https://user-images.githubusercontent.com/70103733/136823918-d266b022-a234-4c0a-ae87-e515d8ae758c.png)

**Comment by TheSuranaverse** [comment]
@Cyril-Mathew join the community & we will help you out
Just reach out to us

A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏
